base_classes/nyaml/NXdata.yaml

```
@@ -237,6 +237,21 @@ NXdata(NXobject):
       .. note::  Attributes potentially containing multiple values
          (axes and _indices) are to be written as string or integer arrays,
          to avoid string parsing in reading applications.
+  \@AXISNAME_depends:
+    doc: |
+      Points to the path to a field defining the axis on which the ``AXISNAME`` axis depends.
+
+      This concept allows to store a link to an axis to a respective field the Nexus hierarchy
+
+      Here, *AXISNAME* is to be replaced by the name of each
+      field described in the ``axes`` attribute.
+
+      Examples:
+          If a calibration has been performed, ``@AXISNAME_depends`` links to the result of
+          that calibration:
+            @AXISNAME_depends: '/entry/process/calibration/calibrated_axis'
+          If the axis is given in a detecotr, ``@AXISNAME_depends`` links to that detector axis:
+            @AXISNAME_depends: '/entry/instrument/detector/AXISNAME' for a 2D detector
```

This field does not exist in NXdetector. I think we discussed to change data in NXdetector into an NXdata group, then the following would apply:
```suggestion
          If the axis corresponds to a coordinate of a detector, ``@AXISNAME_depends`` links to that detector axis:
            @AXISNAME_depends: '/entry/instrument/detector/data/AXISNAME' for a 2D detector
```

---

We should also add a scanned axis:
```suggestion
          If the axis is a scanned motor, ``@AXISNAME_depends`` links to the transformation describing the respective motion, e.g.:
            @AXISNAME_depends: '/entry/instrument/detector/transformations/AXISNAME' for a motion of the detector
```

---

I guess you don't want to replace the lines in red here, but rather add the green lines below. If so, I have done that.